generators/dockertools/templates/python/Dockerfile

```diff
@@ -15,10 +15,19 @@ RUN apk --update add --virtual build-dependencies libffi-dev openssl-dev python-
 
 COPY . /app
 <% if (!enable) { %>
+<% if (language === "PYTHON") { %>
```

Comment:
So flask's language is `PYTHON` and django's language is `DJANGO`? This seems kinda confusing.

Reply:
This sounds like a huge pain in the butt for you to change right now, but it may be worth creating an issue in the future whether we should change `PYTHON` to `FLASK` or if they both need to be `PYTHON`. 

 My concern is that because it's called `DJANGO`, it will mean that it won't generate anything for `generator-readme`. I looked at [Java's choice](https://github.ibm.com/arf/generator-readme/blob/development/generators/app/index.js#L86) and they keep both frameworks as language `JAVA`... so we may need to look further into this.

@Aerex What are your thoughts on this?